src/components/Game/Game.tsx

```diff
@@ -67,50 +65,44 @@ export const Game = () => {
   const onFirstSubmit = async () => {
     setGameStatus(GameStatus.inprogress);
     setCardsForPlayer(await cardsApi.getNewCard(deckId, 2));
+    setCountDealer(await setDealersHand());
   };
 
   const onSubmit = async () => {
```

Comment:
onStartSubmit, onFirstSubmit, onSubmit. As a person trying to figure out how the code works from the names of the functions, it’s not clear to me when they work and it’s very easy to get confused